Source/environment.py

```diff
@@ -0,0 +1,137 @@
+# Author: Siena Okuno
+# Project: Interactive Programming
+# Date: 10.30.2017
+# Description: Creates GUI for player and posts events when a button is clicked
+
+import pygame
+import os
+import sys
+
```

Comment:
I would put all the functions below into a class for the screen. You're changing an object in your game essentially which is when classes are really nice :)